activeresource.gemspec

```diff
@@ -21,8 +21,8 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.2.2'
 
-  s.add_dependency('activesupport', '~> 5.x')
-  s.add_dependency('activemodel',   '~> 5.x')
+  s.add_dependency('activesupport', '< 5.1')
```

Comment:
This line already in activeresource Gemfile. When remove ```s.add_dependency('rails-observers', '~> 0.1.2')``` from gemspec the the correct rails-observers from github is installed. If keep dependency, i need to explicit the gem on my project Gemfile